### ISSUE TITLE
Document fields bypass filterControlCharacters 

### ISSUE THREAD

**Issue by ro0NL** [solariumphp/solarium#953]
Hi,

Today we had a hotfix release because of

https://github.com/solariumphp/solarium/blob/beec496540c3d227201c556729d2c61d1c1f8ab1/src/QueryType/Update/Query/Document.php#L158-L160

which bypasses

https://github.com/solariumphp/solarium/blob/beec496540c3d227201c556729d2c61d1c1f8ab1/src/QueryType/Update/Query/Document.php#L149

why? :D

**Comment by thomascorthals** [comment]
> why? :D

Simple oversight, presumably.

I'll look into getting this fixed ASAP.

**Comment by ro0NL** [comment]
Great. Ref on our side;

```
$ composer show solarium/solarium | grep version
versions : * 4.2.0
````

Updating is on our list, so feel free to pick lowest maintained branch on your side :+1:

**Comment by thomascorthals** [comment]
This was fixed in [Solarium 6.1.6](https://github.com/solariumphp/solarium/releases/tag/6.1.6).
